package.json

```diff
@@ -8,6 +8,7 @@
     "@venly/web3modal": "1.9.5-develop.1",
```

Comment:
shouldn't the web3-provider version be updated too?

Reply:
forgot to remove from package.json since we are importing umd build from cdn